routes/authRoutes.js

```diff
@@ -0,0 +1,42 @@
+const express = require('express')
+const User = require('../models/user')
+
+const app = express()
+app.use(express.json())
+
+// Sign up route
+app.post('/signup', async (req, res) => {
+    const { email, password } = req.body
+
+    try {
+        await User.create({ email, password })
+        res.status(201).send('User created')
+    } catch (err) {
+        res.status(500).send('Internal server error:\n' + err)
+    }
+})
+
+// Login route
+app.post('/login', async (req, res) => {
+    const { email, password } = req.body
+
+    try {
+        const user = await User.findOne({ email })
+
+        if (!user) {
+            return res.status(401).send('Unauthorized')
+        }
+
+        const isMatch = await user.comparePassword(password)
+
+        if (isMatch) {
+            res.status(200).send('Login successful')
```

Comment:
Further down the line but soon we will need to probably send the frontend more information here, like some sort of token that we can re-use to verify that the user is logged in without having to make them login each time. As you say, time will tell.